jujugui/static/gui/src/app/components/deployment-flow/credential/add/add.js

```diff
@@ -48,23 +56,20 @@ YUI.add('deployment-credential-add', function() {
     },
 
     /**
-      Generate a full template object in the expected format.
+      Generate a full credential object in the expected format.
 
-      @method _generateTemplate
+      @method _generateCredentials
```

Comment:
Add `@param id ...` and `@return ...` to the docstring please, both explaining what's received and returned (including a hint to the types, e.g. I'd assume id is a string).